astroquery/utils/tap/core.py

```diff
@@ -207,6 +207,20 @@ def launch_job(self, query, name=None, output_file=None,
                                         "sync",
                                         verbose,
                                         name)
+        # handle redirection
+        if response.status == 303:
+            # redirection
+            if verbose:
+                print("Redirection found")
+            location = self.__connHandler.find_header(
+                response.getheaders(),
+                "location")
+            if location is None:
+                raise Exception("No location found after redirection was received (303)")

```

Comment:
Could you make this a specific exception type?  Perhaps `requests.exceptions.HTTPError`